beaglebone-green.coffee

```diff
@@ -0,0 +1,53 @@
+deviceTypesCommon = require 'resin-device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+BBB_FLASH = 'Power up the <%= TYPE_NAME %> while holding down the small button near the SD slot.
+You need to keep it pressed until the blue LEDs start flashing wildly.'
+
+postProvisioningInstructions = [
+	instructions.BOARD_SHUTDOWN
+	instructions.REMOVE_INSTALL_MEDIA
+	instructions.BOARD_REPOWER
+]
+
+module.exports =
+	slug: 'beaglebone-green'
+	name: 'BeagleBone Green'
+	arch: 'armv7hf'
+	state: 'experimental'
```

Comment:
I think it should pass at least one release testing cycle before changing the experimental flag. But no strong opinion on that and can remove it if we want to do that